tiledb/sm/crypto/test/main.cc

```diff
@@ -24,11 +24,11 @@
  * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
+ *
+ * @section DESCRIPTION
+ *
+ * This file defines a test `main()`
  */
 
-#include "../crypto.h"
-
-int main() {
-  (void)sizeof(tiledb::sm::Crypto);
-  return 0;
-}
+#define CATCH_CONFIG_MAIN
```

Comment:
Since there's only one test file, and there are unlikely to ever be more than that one, this definition could just go in the test source file, which would allow this file to be eliminated.

Reply:
Actually this file is [not needed at all](https://github.com/catchorg/Catch2/blob/devel/docs/migrate-v2-to-v3.md#:~:text=Delete%20TU%20with%20CATCH_CONFIG_RUNNER%20or%20CATCH_CONFIG_MAIN%20defined%2C%20as%20it%20is%20no%20longer%20needed.) and the entry point is provided by the `Catch2::Catch2WithMain` target. I removed it.